app/controllers/api/v2/courses_controller.rb

```diff
@@ -49,11 +52,12 @@ def index
         authorize @provider, :can_list_courses?
         authorize Course
 
-        render jsonapi: @provider.courses, include: params[:include]
+        render jsonapi: @provider.courses, include: params[:include], class: CourseSerializersService.new.execute
       end
 
       def show
-        render jsonapi: @course, include: params[:include]
+        # https://github.com/jsonapi-rb/jsonapi-rails/issues/113
+        render jsonapi: @course, include: params[:include], class: CourseSerializersService.new.execute
```

Comment:
@misaka anything fancy we can do on the jsonapi config

https://github.com/DFE-Digital/manage-courses-backend/blob/f4cedf7c693592147600f4131405b21be343eb9d/config/initializers/jsonapi.rb

```
         render jsonapi: @course, include: params[:include], class: CourseSerializersService.new.execute 
```